apollo-backend/src/main/java/io/logz/apollo/blockers/Blocker.java

```diff
@@ -1,10 +1,6 @@
 package io.logz.apollo.blockers;
 
-/**
- * Created by roiravhon on 6/4/17.
- */
-public class Blocker {
-
+public abstract class Blocker<T extends BlockerFunction> {
```

Comment:
don't need to add `<T extends BlockerFunction>` we work with concrete types in blocker

Reply:
True, forgot to remove it.